src/azure-cli/HISTORY.rst

```diff
@@ -364,7 +364,7 @@ Release History
 **Compute**
 
 * New license type RHEL_ELS_6 (#16012)
-* Adopt track2 SDK, azure-mgmt-compute==18.0.0 (#15750)
+* Adopt track2 SDK, azure-mgmt-compute==18.0.0, including breaking changes such as property name change (#15750)
```

Comment:
Shouldn't we explicitly call out what BREAKING CHANGES are introduced?

Reply:
Done